### PR TITLE
Expand msgpack wrapper code

### PR DESCRIPTION
**Pull request by altran01**
This is required to handle more complicated situations like dis code

### CODE REVIEW

File: python/veles/schema/nodeid.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return self._bytes
+
+    def to_bytes(self):
+        return self.bytes
+
+    def __str__(self):
+        return binascii.b2a_hex(self.bytes).decode('ascii')
+
+    def __repr__(self):
+        return 'ObjectID("{}")'.format(str(self))
```
Comment (koriakin) [inline python/veles/schema/nodeid.py]: error

File: python/veles/asrv/proto.py [hunk 2]
```diff
@@ -96,7 +98,10 @@ def data_received(self, data):
         self.unpacker.feed(data)
         while True:
             try:
-                msg = definitions.MsgpackMsg.load(self.unpacker)
+                msg = definitions.MsgpackMsg.load(self.unpacker.unpack())
+                if not isinstance(msg, definitions.MsgpackMsg):
```
Comment (koriakin) [inline python/veles/asrv/proto.py]: This shouldn't be needed.

File: python/veles/messages/definitions.py [hunk 3]
```diff
@@ -12,45 +12,17 @@
...
-            obj[field.name] = field.__get__(self)
-        return packer.pack(obj)
-
-    def __str__(self):
-        msg = '{}: {}'.format(self.msg_type, self.to_dict())
-        return msg
-
-    __repr__ = __str__
+class MsgpackMsg(model.PolymorphicModel):
+    object_type = None
```
Comment (koriakin) [inline python/veles/messages/definitions.py]: object_type shouldn't be needed

**Comment by koriakin** [comment]
It would be good to split off the ObjectID -> NodeID rename - that can be merged immediately.

File: python/veles/messages/definitions.py (hunk 3 above)
Comment (altran01) [inline python/veles/messages/definitions.py]: It wasn't.

**Pull request merged**